src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.html

```diff
@@ -1,136 +1,207 @@
+                  </label>
+                  <div class="cd-col-form-input">
+                    <input class="form-control"
+                           id="walSlots"
+                           name="walSlots"
+                           type="number"
+                           min="0"
+                           formControlName="walSlots">
+                    <span class="invalid-feedback"
+                          *ngIf="form.showError('walSlots', formDir, 'min')"
+                          i18n>Value should be greater than or equal to 0</span>
+                  </div>
+                </div>
 
-          <!-- DB devices button and table -->
-          <cd-osd-devices-selection-groups #dbDeviceSelectionGroups
-                                           name="DB"
-                                           type="db"
-                                           [availDevices]="availDevices"
-                                           [canSelect]="dataDeviceSelectionGroups.devices.length !== 0"
-                                           (selected)="onDevicesSelected($event)"
-                                           (cleared)="onDevicesCleared($event)">
-          </cd-osd-devices-selection-groups>
+                <!-- DB devices button and table -->
+                <cd-osd-devices-selection-groups #dbDeviceSelectionGroups
+                                                 name="DB"
+                                                 type="db"
+                                                 [availDevices]="availDevices"
+                                                 [canSelect]="dataDeviceSelectionGroups.devices.length !== 0"
+                                                 (selected)="onDevicesSelected($event)"
+                                                 (cleared)="onDevicesCleared($event)">
+                </cd-osd-devices-selection-groups>
 
-          <!-- DB slots -->
-          <div class="form-group row"
-               *ngIf="dbDeviceSelectionGroups.devices.length !== 0">
-            <label class="cd-col-form-label"
-                   for="dbSlots">
-              <ng-container i18n>DB slots</ng-container>
-              <cd-helper>
-                <span i18n>How many OSDs per DB device.</span>
-                <br>
-                <span i18n>Specify 0 to let Orchestrator backend decide it.</span>
-              </cd-helper>
-            </label>
-            <div class="cd-col-form-input">
-              <input class="form-control"
-                     id="dbSlots"
-                     name="dbSlots"
-                     type="number"
-                     min="0"
-                     formControlName="dbSlots">
-              <span class="invalid-feedback"
-                    *ngIf="form.showError('dbSlots', formDir, 'min')"
-                    i18n>Value should be greater than or equal to 0</span>
-            </div>
-          </div>
-        </fieldset>
+                <!-- DB slots -->
```

Comment:
@avanthakkar Those are indendation changes to proper align it and it was not a mistake. If I remove it it will be misaligned with the above and below lines.

Reply:
@avanthakkar Those are indendation changes to proper align it and it was not a mistake. If I remove it it will be misaligned with the above and below lines.